types/header.go

```diff
@@ -77,6 +176,15 @@ func (n Nonce) MarshalText() ([]byte, error) {
 	return []byte(n.String()), nil
 }
 
+func (n *Nonce) UnmarshalText(input []byte) error {
+	input = input[2:]
```

Comment:
What is this magic number `2`?

Reply:
Why is this an exported method?

Reply:
I saw that we have pattern [2:] in another package also in the future maybe i am going to need this somewhere outside of this package, that's why i left it exported